docs/getting-started.md

```diff
@@ -20,32 +25,35 @@ You will not be running this very often. Typically you'll be building a new imag
 2. `docker-compose`
 This command is responsible to "spin-up" the container that is defined in `docker-compose.yml`. This file essentially tells Docker to run an instance of the image that was previously built. At this step, we will specify the environment-specific configurations as well as "mount" volumes that are shared by both your local machine as well as the docker container. This is useful because you may want to change some code in your local machine and quickly test those changes inside the docker container to see if it had the intended effect.
 
+### Set Up
+
+**Clone this Git repository**
 
-### Pre-requisites
-- Create a local git repository
 ```
-mkdir your-dir-name
-cd your-dir-name
-git init
-git remote add origin git@github.com:nyc-cto/tms.git
-git pull origin master
+git clone git@github.com:nyc-cto/tms.git
+cd tms
 ```
 
-- Docker Desktop (https://www.docker.com/get-started)
+**Set up env.template**
+
 - Copy over `.env.template` into `.env` with any other environment variables the application needs access to. Whenever you update `.env`, don't forget to `source .env`
-- Setup Secrets by looking at the following templates and re-creating them:
-	- `git_key.template` -> `git_key`
-	- `IngestionGoogleKey.json.template` -> `IngestionGoogleKey.json`
-	- `TranslationGoogleKey.json.template` -> `TranslationGoogleKey.json`
+
+**Set up secrets files**
+
+```bash
+cp secrets/git_key_template git_key
+cp secrets/IngestionGoogleKey.json.template IngestionGoogleKey.json
+cp TranslationGoogleKey.json.template TranslationGoogleKey.json
+```
```

Comment:
Can you clarify that the person needs to read over the files and edit them as appropriate, not just copy them?